HomeSeer/homeseer-zwave-switch/fingerprints.yml

```
@@ -17,6 +17,36 @@ zwaveManufacturer:
     productType: 0x4447
     productId: 0x3034
     deviceProfileName: homeseer-wd100
+  - id: "000C/4447/3033"
+    deviceLabel: HomeSeer WS100+ Switch
+    manufacturerId: 0x000C
+    productType: 0x4447
+    productId: 0x3033
+    deviceProfileName: homeseer-ws100
+  - id: "0315/4447/3036"
+    deviceLabel: HomeSeer WD200+ Dimmer - ZWaveProducts.com
+    manufacturerId: 0x0315
+    productType: 0x4447
+    productId: 0x3036
+    deviceProfileName: homeseer-wd200zwp
```

`fingerprints.yml` just [binds a Device Type to a Device Profile](https://community.smartthings.com/t/tutorial-creating-drivers-for-zigbee-devices-with-smartthings-edge/229502#fingerprintsyaml-9). The `deviceProfileName` has to match a profile in the profiles directory. The re-branding shouldn't change the profile at all, because it's the same switch. So the ZWaveProducts and HomeSeer fingerprints should be bound to the same profiles.
```suggestion
    deviceProfileName: homeseer-wd200
```

---

Also, the `ws100` needs a profile file in the profiles directory.

---

Thanks! Added ws100 profile, and corrected the profile names.